packages/office-ui-fabric-react/src/components/ContextualMenu/ContextualMenu.base.tsx

```diff
@@ -136,6 +140,13 @@ export class ContextualMenuBase extends BaseComponent<IContextualMenuProps, ICon
           : null;
       }
     }
+    if (newProps.delayUpdateFocusOnHover !== this.props.delayUpdateFocusOnHover) {
```

Comment:
>if (newProps.delayUpdateFocusOnHover !== this.props.delayUpdateFocusOnHover) { [](start = 4, length = 78)

is the complexity introduced by delayUpdateFocusOnHover and _shouldUpdateFocusOnMouseEvent actually needed? Windows does not seem to do that and if the user moves the mouse at that point the user is mixing\switching input times so it that bad that focus moves? Seems that the complexity introduced here might not be worth the benefit?

Reply:
The TellMe control is exactly why this was added, it does a mix of keep focus in an input with the menu expanded and also moving focus from the input to the menu

Reply:
More generally, if someone creates a contextualMenu with shouldFocusOnMount = false, they may not want hover to update focus to the menu until they put focus in the menu

Reply:
talked online, sounds good, I forgot about tellMe scenarios

---
In reply to: [213418687](https://github.com/OfficeDev/office-ui-fabric-react/pull/6109#discussion_r213418687) [](ancestors = 213418687)